wrappers/unity/Assets/RealSenseSDK2.0/Materials/BGSegMat.mat

```diff
@@ -0,0 +1,102 @@
+%YAML 1.1
```

Comment:
Can you add documentation/description why you had to modify default settings?

Reply:
which default settings?